src/org/pcic/GitUtils.groovy

```diff
@@ -8,14 +8,14 @@ class GitUtils implements Serializable {
         this.steps = steps
     }
 
-    ArrayList isCommitTagged() {
-        String gitTags = steps.sh(script: 'git tag --contains', returnStdout: true)
+    ArrayList getCommitTags() {
+        String tags = steps.sh(script: 'git tag --contains', returnStdout: true)
 
-        if (gitTags.isEmpty()) {
+        if (tags.isEmpty()) {
```

Comment:
Sorry, I have to ask: Doesn't `tags.trim().split('\n')` work properly when `tags.isEmpty()`?

Reply:
Unfortunately in `groovy` an error is thrown if you run `.trim()` on an empty string.